_data/accessibility-tests/character-count.yml

```diff
@@ -0,0 +1,68 @@
+    summary_additional: |
+      The text field has a visible label above it to indicate its purpose. For example, "Enter your full name."
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.3.5
+  - summary: Formatting remains when screen is magnified.
+    summary_additional: |
+      When you zoom in to 200%,
+      the character count keeps its formatting, and sufficient space remains between the character count and the text field above it.
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.4.12
+  - summary: Users can see and hear context about character count errors.
+    summary_additional: |
+      Clear, readable information about character count appears below the text field.
+      When you use a screen reader to tab to the text field, you hear accurate information about how many characters remain for the user to enter.
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 3.3.2
+# Zoom/screen magnification tests
+  - summary: Content remains visible and functional when screen is magnified.
+    summary_additional: |
+      When you zoom in to 200%,
+      text stays in position inside the text field, and the character count keeps its position below the text field.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.7.1
+    wcag_criterion: 1.4.4
+# Keyboard navigation tests
+  - summary: Field can be accessed with keyboard navigation.
+    summary_additional: Using only a keyboard, you are able to tab into the character count's text input field, type, and tab out of it.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.7.1
+    wcag_criterion: 2.1.1
+# Screen reader tests
+  - summary: Screen reader announces field and character count.
+    summary_additional: |
+      When you tab into the text field with a screen reader,
+      it announces the purpose of the field and the presence of a character count.
+      For example, "This is a text input with a character count."
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.7.1
+    wcag_criterion: 2.4.6
+  - summary: Screen reader announces violations of word limit.
+    summary_additional: After you enter one too many characters, the screen reader announces that you have exceeded the character limit.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.7.1
+    wcag_criterion: 3.3.1
+  - summary: Users receive feedback about character count errors.
+    summary_additional: |
+      After you enter too many characters, the screen reader states how many characters over the limit you've gone.
+      Red letters under the text field also show the how many characters over the limit you have entered.
+      Feedback about how many characters need to be removed can be heard and seen.
+    test_status: exception
```

Comment:
I think it is because we do not provide a way to tell the user **how** they need to fix the error. While we do identify the error for the user of how many characters they have gone over, it does not specifically say something like, "Ensure you have X number of characters". I think saying that you have gone over is a start to suggesting, which is why I think it is in warn and not pass. @amyleadem

Reply:
The current explanatory message about why it passed with exceptions reads, "The character count accessibility tests component may not pass all elements of this test." @amyleadem, @alex-hull, @amycole501, and @finekatie - I think we need another sentence or two to explain why. Maybe something like, "The character count accessibility tests component may not pass all elements of this test. The character count informs screen reader users of how many characters over the limit they have entered, but it does not explicitly tell them the character limit they need to stay under." What do you think?

And is there a GitHub issue to link to, too?

Reply:
I don't think there's an issue submitted; at least, I didn't submit one. @amyleadem should I submit an issue to ensure the character count code states the limit explicitly?

Reply:
@amycole501 @alex-hull for clarity, can you answer the following:
1. Is this an item we want to explicitly mark as "passed with exceptions"? 
2. If so, is there a related uswds issue for this?  
3. If so, is this an appropriate description for the issue: "The character count accessibility tests component may not pass all elements of this test. The character count informs screen reader users of how many characters over the limit they have entered, but it does not explicitly tell them the character limit they need to stay under."

Reply:
Whoops, I didn't hit refresh before submitting these questions @amycole501. Looks like you already answered one of them :)

If you could submit an issue and send me an issue number, that would be great!

Reply:
Issue is here for character count error suggestion/prevention https://github.com/uswds/uswds/issues/5873

Your wording is great!

I do think this passes with exceptions since the instructions disappear once you start typing.

Reply:
Updated in [3651fdc](https://github.com/uswds/uswds-site/pull/2607/commits/3651fdc3de1bf9f4358e295c33817a5f928b559d). Let me know if you need any edits.